cmd/gossamer/configcmd_test.go

```diff
@@ -58,13 +58,13 @@ var TestGenesis = &genesis.Genesis{
 
 func teardown(tempFile *os.File) {
 	if err := os.Remove(tempFile.Name()); err != nil {
-		log.Warn("cannot remove temp file", "err", err)
+		log.Warn("[gossamer] cannot remove temp file", "err", err)
```

Comment:
idk why we are logging in the test file, change this to `t.Log()` or remove?

Reply:
can do, was not thinking about why the log was there, just updated the format